README.md

```diff
@@ -73,15 +72,6 @@ This theme uses a very high loadorder of 1500 in order to be able to overwrite m
 - If you pay close attention to the console log you might find [Warning] entries related to this theme.
 - Celestians "Author" extension currently makes storyframe list entries hard to read (black). I'm working on resolving the issue. For now, don't use my theme during world building with the Author extension enabled.
 
-### Want to buy me a coffee?
-[PayPal War Chest](https://www.paypal.com/paypalme/DnDKriegskasse)
-or
-[Ko.Fi](https://ko-fi.com/sirmotte)
-
-### Want to buy me a DMs Guild gift instead?
-Here's my [Dungeon Masters Guild Maps Wishlist](https://www.dmsguild.com/wishlist_public.php?public_id=2593421&list_id=3816273&buying_for=sirmotte%40posteo.de&quot;]Wishlist%20for%20Maps), or my general [PDF Wishlist](https://www.dmsguild.com/wishlist_public.php?public_id=2593421&list_id=3869339&buying_for=sirmotte%40posteo.de)
-
-
```

Comment:
Looks like you did some readme cleanup... if not your changes, just alter my PR.